units/src/iosMain/kotlin/dev/icerock/moko/units/UnitCollectionViewDataSource.kt

```diff
@@ -13,18 +13,27 @@ import platform.UIKit.row
 import platform.darwin.NSInteger
 import platform.darwin.NSObject
 
+typealias UICollectionViewReloadHandler = (UICollectionView, oldData: List<CollectionUnitItem>?, newData: List<CollectionUnitItem>?) -> Unit
+
 @ExportObjCClass
-class UnitCollectionViewDataSource(
-    private val collectionView: UICollectionView
-) : NSObject(), UICollectionViewDataSourceProtocol {
+class UnitCollectionViewDataSource internal constructor(
+    private val collectionView: UICollectionView,
+    private val reloadDataHandler: UICollectionViewReloadHandler
```

Comment:
here need default value, to simple usage from kotlin side

Reply:
just move from here `withReloadHandler: UICollectionViewReloadHandler = { collectionView, _, _ -> collectionView.reloadData() }`

Reply:
done